docs/rules-engine/examples/runtime-facts.md

```diff
@@ -1,92 +1,431 @@
+                "key": "welcome-text",
+                "value": "welcome-text-with-promotion"
               }
-            ],
-            'failureElements': [
+            ]
+          }
+        },
+        {
+          "id": "5467e501-b9ff-414f-8026-56885d0d7a4b",
+          "name": "Teaser for minors in Paris",
+          "disabled": false,
+          "outputRuntimeFacts": [],
+          "inputRuntimeFacts": [],
+          "inputFacts": [
+            "ageOfCustomer",
+            "locationOfConference"
+          ],
+          "rootElement": {
+            "elementType": "RULE_BLOCK",
+            "blockType": "IF_ELSE",
+            "condition": {
+              "any": [
+                // The customer is a minor in France
+                {
+                  "all": [
+                    {
+                      "lhs": {
+                        "type": "FACT",
+                        "value": "ageOfCustomer"
+                      },
+                      "operator": "lessThan",
+                      "rhs": {
+                        "type": "LITERAL",
+                        "value": 18
+                      }
+                    },
+                    {
+                      "lhs": {
+                        "type": "FACT",
+                        "value": "locationOfConference"
+                      },
+                      "operator": "equals",
+                      "rhs": {
+                        "type": "LITERAL",
+                        "value": "Paris"
+                      }
+                    },
+                  ],
+                },
+                // The customer is a minor in the USA
+                {
+                  "all": [
+                    {
+                      "lhs": {
+                        "type": "FACT",
+                        "value": "ageOfCustomer"
+                      },
+                      "operator": "lessThan",
+                      "rhs": {
+                        "type": "LITERAL",
+                        "value": 21
+                      }
+                    },
+                    {
+                      "lhs": {
+                        "type": "FACT",
+                        "value": "locationOfConference"
+                      },
+                      "operator": "equals",
+                      "rhs": {
+                        "type": "LITERAL",
+                        "value": "New York"
+                      }
+                    }
+                  ]
+                }
+              ],
+              "successElements": [
+                {
+                  "actionType": "UPDATE_PLACEHOLDER",
+                  "key": "teaser-placeholder",
+                  "value": "assets/placeholders/teaser-activities.json"
+                }
+              ]
+            }
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+
+In this Ruleset, there is some duplication over the logic of "the customer is a minor". This makes it more difficult to
```

Comment:
Instead can't we avoid providing wrong example and directly say 
"to avoid duplication and maintenance complexity we can go this way"
and show only the way to go